scripts/update_off_count.py

```
@@ -52,19 +53,67 @@ def compute_total() -> int:
+            off["metadata"]["sample_size"] = total
+
+        # Recalculate checksum after updating counts
+        off["checksum"] = calculate_checksum(off)
+
+        meta["openfoodfacts"] = off
+        VERS.write_text(json.dumps(meta, ensure_ascii=False, indent=2), encoding="utf-8")
+
+        print(f"OFF record_count updated from {original_record_count} to {total}")
+        print(f"Checksum recalculated: {off['checksum'][:16]}...")
+
+        return 0
```

Ruff (0.13.1)

Consider moving this statement to an `else` block

(TRY300)